leaderboard/web/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  backendUrl: 'http://devopsoh2subproc1fun/api/leaderboard/',
+  backendUrl: 'http://localhost:8080/api/leaderboard/',

```

Comment:
You need to keep it this value OR [change this line](https://github.com/Azure-Samples/openhack-devops-proctor/blob/65939ec4b604a14225f5aa52fa18dc0fc6a67601/leaderboard/web/Dockerfile#L6) to replace `localhost:8080`.  Otherwise it will break the website.

Reply:
Did the change